### PR TITLE
Profile page and Image upload page, youtube page

### PR DESCRIPTION
**Pull request by jamierashley**
### What is a quick description of the change?

Profile page and image upload page designs changed to match new home page,
alert button added to javascript to let user know color data has been recieved
youtube videos displayed with title underneath (didn't catch a ss before quota was exceeded)

### Screenshot/Screengrab after this change?
Screenshot 2020-07-28 at 10 49 00 AM
Screenshot 2020-07-28 at 11 14 11 AM
Screenshot 2020-07-28 at 10 48 52 AM

### Is this fixing an issue?

none

### Are there more details that are relevant?

color of purple maybe slightly off

### CODE REVIEW

File: src/main/webapp/quiz.html [hunk 1]
```diff
@@ -62,6 +62,10 @@ <h2 id="quiz-title">Style Quiz</h2>
             <button id="next">Next</button>
 
             <!-- Submit Quiz Button -->
+            <!-- <form id="answers">
```
Comment (wu-haotian) [inline src/main/webapp/quiz.html:65]: This seems unrelated to your PR?

Comment (jamierashley) [reply src/main/webapp/quiz.html:65]: Oh I think I committed the on accident. It's no different than what's already in the repo.

**Review by aliciasimon — APPROVED** [review]
LGTM
